pycyphal/application/_node.py

```diff
@@ -128,10 +133,14 @@ def make_publisher(self, dtype: Type[T], port_name: str | int = "") -> Publisher
         The type information is automatically exposed via ``uavcan.pub.PORT_NAME.type`` based on dtype.
         For details on the standard registers see Specification.
 
-        The port_name may also be the integer port-ID if it is desired to bypass the registry (not recommended).
+        **Experimental:** the ``port_name`` may also be the integer port-ID.
+        In this case, new port registers will be created with the names derived from the supplied port-ID
+        (e.g., ``uavcan.pub.1234.id``, ``uavcan.pub.1234.type``).
+        If the ID register created in such way is overridden externally,
```

Comment:
Your wording changes the meaning.

Say, when I do `node.make_publisher(namespace.MyType_1_0, 1234)` for the first time, PyCyphal will create a new pair of registers:

- `uavcan.pub.1234.id` containing `natural16 = [1234]`
- `uavcan.pub.1234.dtype` containing `string = "namespace.MyType.1.0"`

Then someone might assign `uavcan.pub.1234.id` = 7654. If the register is persistent, at the next launch your 1234 will be ignored in favor of 7654.

The objective is to minimize the chances that someone will decide to hard-code all port-IDs in the application.

Reply:
Okay. You are missing an `a`:

```
If the ID register created in such a way is overridden externally,
```

but I think this would be clearer:

```
If ID registers created this way are overridden externally,
```

Reply:
Sorry, this got merged automatically. I am going to fix this on the development branch.